exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -78,14 +91,50 @@ func (e *exporter) PushMetricsData(ctx context.Context, md pdata.Metrics) error
 	return nil
 }
 
-func (e *exporter) serializeMetrics(md pdata.Metrics) ([]string, int) {
+func dimensionsFromTags(tags []string) dimensions.NormalizedDimensionList {
+	dims := []dimensions.Dimension{}
+	for _, tag := range tags {
+		parts := strings.SplitN(tag, "=", 2)
+		if len(parts) == 2 {
+			dims = append(dims, dimensions.NewDimension(parts[0], parts[1]))
+		}
+	}
+	return dimensions.NewNormalizedDimensionList(dims...)
+}
+
+func labelsToDimensions(labels pdata.StringMap) []dimensions.Dimension {
+	dims := []dimensions.Dimension{}
+	labels.Range(func(k, v string) bool {
+		dims = append(dims, dimensions.NewDimension(k, v))
+		return true
```

Comment:
Does the OTel collector `labels.Range` method require the passed function to return bool? If so, do you know why?

Reply:
Returning `false` stops the iteration. You have to return `true` to iterate over every item.